rpc/src/p2p.rs

```diff
@@ -1,17 +1,67 @@
+use celestia_types::p2p::{
+    AddrInfo, BandwidthStats, Connectedness, PeerId, Reachability, ResourceManagerStats,
+};
 use jsonrpsee::proc_macros::rpc;
-use serde::{Deserialize, Serialize};
-
-#[derive(Debug, Serialize, Deserialize)]
-pub struct AddrInfo {
-    #[serde(rename = "ID")]
-    pub id: String,
-    // TODO: multiaddr
-    #[serde(rename = "Addrs")]
-    pub addrs: Vec<String>,
-}
 
 #[rpc(client)]
 pub trait P2P {
+    #[method(name = "p2p.BandwidthForPeer")]
+    async fn p2p_bandwidth_for_peer(&self, peer_id: &PeerId) -> Result<BandwidthStats, Error>;
```

Comment:
If you notice in other modules (for example in blob) we have as doc-comments the [official description](https://node-rpc-docs.celestia.org/#p2p) of each function. Just copy-paste it, no need to adjust anything.

Reply:
Copied official documentation into doc-comments for appropriate methods.